web/cgi-bin/horas/specials.pl

```diff
@@ -1367,25 +1367,29 @@ sub oratio {
 
 sub getind {
   my ($office, $num, $day) = @_;
+
   if ($office !~ /Sancti/i) { return $num; }
   if ($hora =~ /Laudes/i) { return 2; }
   my $d = sprintf("-%02i", $day);
-  if ($office =~ /$d.*\.txt/i) { return 3; }
+  if (concurrent_office($office,$num,$day)) { return 3; }
   return 1;
 }
 
 #*** concurrent_office($office, $num, $day)
 #returns 1 if the office is concurrent
 sub concurrent_office {
   my ($office, $num, $day) = @_;
-  my $d = sprintf("%02i", $day);
+  my $office_mmdd = $1 if $office =~ /([0-9]+-[0-9]+)/;
```

Comment:
This pattern matches more than just sanctoral offices.  For example, the Ember days in September are `093-[356]`.  I haven't thought through whether this matters in practice, but I'd be in favour of tightening up the regex.